ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -6,6 +6,15 @@
 extern crate log;
 
 #[cfg(target_os = "windows")]
+use libc;
```

Comment:
This won't work - `#[cfg]` only applies to the immediately following statement. This is why I suggested moving all of the new `use` lines inside the new function that makes use of them.

Reply:
Ok.  Moved the "use …" statements as shown below and removed libc.
fn do_redirect_stdout_stderr() -> Result<(), ()> {
    use winapi::shared;
    use winapi::um::debugapi;
    use winapi::um::handleapi;
    use winapi::um::minwinbase;
    use winapi::um::namedpipeapi;
    use winapi::um::processenv;
    use winapi::um::winbase;
    use winapi::um::winnt;